docs/topics/loaders.rst

```diff
@@ -20,6 +20,10 @@ Item Loaders are designed to provide a flexible, efficient and easy mechanism
 for extending and overriding different field parsing rules, either by spider,
 or by source format (HTML, XML, etc) without becoming a nightmare to maintain.
 
+.. note:: Item Loaders are an extension of the itemloaders_ library that make it
+    easier to work with Scrapy by adding support for :ref:`items <topics-items>`
```

Comment:
Aren't items supported in itemloaders out of box, via itemadapter?

Reply:
Yup, I'll leave only the reference to `response`